provisioning/miniprov/hare_mp/main.py

```diff
@@ -222,6 +223,7 @@ def _start_consul(utils: Utils,
 
     provider = ConfStoreProvider(url)
     consul_endpoints = provider.get('cortx>external>consul>endpoints')
+    hostname = utils.get_local_hostname()
```

Comment:
Since you have imported socket library, we can use another function 'socket.gethostname()'.
Just a suggestion.

Reply:
sorry, what difference would it make?

Reply:
No, difference, but it will use standard library functions.